src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -42,6 +42,10 @@ public class MvcRazorHost : RazorEngineHost, IMvcRazorHost
             {
                 LookupText = "Microsoft.AspNet.Mvc.Razor.TagHelpers.UrlResolutionTagHelper, Microsoft.AspNet.Mvc.Razor"
             },
+            new AddTagHelperChunk
```

Comment:
@pranavkm @Eilon if we do this, `@Html.CheckBox()` and `@Html.CheckBoxFor()` will not work correctly inside a regular `<form>` element.